net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd.inc

```diff
@@ -294,25 +294,26 @@ function quagga_ospfd_install_conf() {
 	    && !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd'])) {
 	    // if there is a raw config specified in the config.xml use that instead of the assisted config
 	    $bgpdconffile = str_replace("\r","",base64_decode($config['installedpackages']['quaggaospfdraw']['config'][0]['bgpd']));
-	} else {
-	    $bgpdconffile = "Use vtysh to modify content\n\n";
 	}
+	else 
+	    $bgpdconffile = "";
+
 	$fd = fopen("{$quagga_config_base}/bgpd.conf", "w");
 	fwrite($fd, $bgpdconffile);
 	fclose($fd);
 
         /* Make ospf6 config */
-        if (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d'])
-            && !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d'])) {
-            // if there is a raw config specified in the config.xml use that instead of the assisted config
-            $ospf6dconffile = str_replace("\r","",base64_decode($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d']));
-        } else {
-            $ospf6dconffile = "Use vtysh to modify content\n\n";
-        }
-        $fd = fopen("{$quagga_config_base}/ospf6d.conf", "w");
-        fwrite($fd, $ospf6dconffile);
-        fclose($fd);
+	if (isset($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d'])
+	    && !empty($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d'])) {
+	    // if there is a raw config specified in the config.xml use that instead of the assisted config
+	    $ospf6dconffile = str_replace("\r","",base64_decode($config['installedpackages']['quaggaospfdraw']['config'][0]['ospf6d']));
+	}
+	else
```

Comment:
Need to fix the braces here, too: https://doc.pfsense.org/index.php/Developer_Style_Guide#Indent_style